assets/css/index.css

```
@@ -411,22 +433,25 @@ img {
 	width: 20px;
 	height: 20px;
 }
+
 .itemType {
 	font-style: italic;
 	margin-top: 5px;
 	margin: top 8px;
```

Stylelint (17.13.0)

[error] Overridden property "margin-top" by shorthand "margin" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)